sheets/coupon_assign_api.py

```diff
@@ -364,7 +365,7 @@ def find_bulk_assignment_messages(assignment_status_map, earliest_message_date=N
         assignment_status_map.add_potential_event_date(
             message.bulk_assignment_id,
             message.coupon_code,
-            message.email.lower(),
+            message.email,
```

Comment:
Was this already lowercase?

Reply:
No, I just moved the responsibility to the `add_potential_event_date` method. Made more sense to me that the `AssignmentStatusMap` class would know that it needs to coerce email addresses to lowercase in order to match against emails in one of its own properties